### ISSUE TITLE
Running Buzz 1.2.0 on macOS 12.7 Monterey (2017 MacBook Air, Intel Core)

### ISSUE THREAD

**Issue by mktnag** [chidiwilliams/buzz#826]
Hi,

I successfully installed Buzz 1.0, replacing the previous version 0.8.7 using the dmg file. However, I am unable to run the application on my 2017 MacBook Air, which is running macOS 12.7 Monterey on an Intel Core processor.

Could you please provide assistance or any troubleshooting steps to resolve this issue?

Thank you.

**Comment by chidiwilliams** [comment]
Taking a look...

**Comment by chidiwilliams** [comment]
Do you get a crash report when the app fails? Could you share it if so? If not, could you share the contents of the log file at `/Users/chidiwilliams/Library/Logs/Buzz/logs.txt`?

**Comment by mktnag** [comment]
When I try to open Buzz, whose icon is crossed out with a stop sign, I receive the message from macOS: "You can’t open the application 'Buzz' because this application is not supported on this Mac." There is nothing related to this in the logs.txt file.

**Comment by mktnag** [comment]
![Screenshot 2024-07-08 at 08 25 10](https://github.com/chidiwilliams/buzz/assets/166207126/6f4c2aff-dae4-48c4-9147-f26e38154782)

![Screenshot 2024-07-08 at 08 25 32](https://github.com/chidiwilliams/buzz/assets/166207126/84c1d640-7ae5-4c99-9aee-1b0a08d4ea90)

**Comment by shawkynasr** [comment]
same issue with 2018, MacOS 14.5

**Comment by mktnag** [comment]
Hi, the same issue remains with Buzz 1.0.1; still unable to run on 2017 Macbook

**Comment by Steelpan** [comment]
Like you, @mktnag, I upgraded from a previous version and it crashed when I started it but it runs fine after deleting:
```
/Users/<username>/Library/Caches/Buzz
/Users/<username>/Library/Application Support/Buzz
```
I am running this on an M series MacBook however so your milage may vary.

**Comment by raivisdejus** [comment]
Please get the latest development build for your architecture from here https://github.com/chidiwilliams/buzz/actions/runs/10415617701 (or any [latest](https://github.com/chidiwilliams/buzz/actions?query=branch%3Amain) successful build)
* Intel Macs - `Buzz-macOS-X64`
* Apple silicon Macs - `Buzz-macOS-ARM64`

You need to log into the GitHub, to see the download button

**Comment by mktnag** [comment]
Thank you @raivisdejus it works however the only model available for download seems to be Whisper?

![Screenshot 2024-08-16 at 15 16 18](https://github.com/user-attachments/assets/5efa7e6c-4241-402d-92ea-2ea8edd5a47e)

**Comment by raivisdejus** [comment]
@mktnag Faster whisper does not work on macOS, this is expected. But it looks like also Whisper.cpp failed to load.  I assume it is Intel Mac, right?

**Comment by mktnag** [comment]
Yes Intel Mac. I used to be able to download and use all models on 0.8.7

**Comment by raivisdejus** [comment]
@mktnag Intel version of the app is build on macOS 13, if you are still on 12.7, that could be a reason why the Whisper.cpp is not available. 

In this case I think the best way for you to get Buzz running is to run it from source. 
Get `brew` from here https://brew.sh/ if you do not have it already and see notes on running Buzz from source here https://github.com/chidiwilliams/buzz/blob/main/CONTRIBUTING.md#mac
If it seems complicate, don't worry it is quite strait forward and if something goes wrong or is not clear post here, I will gladly help you to sort it out.

Faster whisper would not be of much use on macOS as it would be slower than Whisper.cpp, but quality is essentially the same.

**Comment by mktnag** [comment]
I was able to do it, but each time I launch it from source, I get an additional copy double of previous transcriptions

**Comment by mktnag** [comment]
![Screenshot 2024-08-19 at 10 42 15](https://github.com/user-attachments/assets/35944853-efa1-46b6-ac3a-4ba6d44f0e26)
I get this error when I try to run Whisper.cpp

**Comment by raivisdejus** [comment]
@mktnag Regarding previous transcripts, pull the latest version of source, this issue is fixed.

Regarding `ffmpeg` is it installed on your system? Like if you run `ffmpeg` in the terminal does it print anything?
If not run `brew install ffmpeg`

Also you can try to install or re-install ffmpeg 
```
pip3 unstall ffmpeg ffmpeg-python  
pip3 install ffmpeg
pip3 install ffmpeg-python
``` 

Please let me know if any of these steps helped, so I can update the notes.

**Comment by mktnag** [comment]
@raivisdejus yes it did help, thank you very much. I am able to run transcriptions successfully now using whisper.cpp on my OSX 12.7.6/Intel (which I cannot upgrade to 13). Would it be possible to have this run normally from a dmg in a future update?

**Comment by raivisdejus** [comment]
Yes, future releases should have Intel Mac dmg files in the release section.

**Comment by raivisdejus** [comment]
For Apple silicon please try the latest version 1.1.0
For Intel get the version from latest development build (you need to log into the github to see the artifact download button) https://github.com/chidiwilliams/buzz/actions/workflows/ci.yml?query=branch%3Amain

**Comment by rizhenkov** [comment]
I have installed version from here https://github.com/chidiwilliams/buzz/actions/runs/11313311335 
and this working fine on my Intel Mac. But versions from releases and from brew does not. 

So, as a temporary solution it is possible to download specific version from build artefacts. But it will be nice if @chidiwilliams also add fresh builds for x64 Macs to releases and brew too.

**Comment by mktnag** [comment]
Hello, thank you @chidiwilliams for the latest update. Could you please confirm if the unzipped Buzz-1.2.0-mac.dmg will work on macOS 12.7 Monterey? I am using a 2017 MacBook Air with an Intel Core processor, which I cannot update to a newer macOS version.

I’m asking because installing my current version from source was quite complex, and I want to avoid uninstalling it unless I’m certain the new version will work on my setup.

Looking forward to your clarification.

**Comment by mktnag** [comment]
Hi, @raivisdejus Until now, I had been using version 1.0.1 of Buzz, which I downloaded from source, and was able to use Whisper.cpp (medium size) without any issues.

I recently tried installing version 1.2.0 from the .dmg on macOS 12, which I simply downloaded from here on Github. While the app installs and opens correctly, I am encountering the same issue I had with version 1.0.1 when using the .dmg installer: the app only allows me to download Whisper, but not Whisper.cpp, which is the version I require.

Is there any way to resolve this? Thank you

**Comment by raivisdejus** [comment]
Moving this issue to https://github.com/chidiwilliams/buzz/issues/1075 that will track all issues for App store MacOS version
